packages/koa-performance/src/middleware.ts

```diff
@@ -126,9 +126,14 @@ export function clientPerformanceMetrics({
             ? event.duration
             : event.start;
 
+        const roundedValue =
+          event.type === EventType.CumulativeLayoutShift
+            ? value
+            : Math.round(value);
+
         metrics.push({
           name: eventMetricName(event),
-          value: Math.round(value),
+          value: roundedValue,
```

Comment:
The reason I added this is because CLS is calculated as a fraction (e.g. 0.1, 0.25) - the rounding here was rounding all CLS values down to 0.

This is not a super elegant way of handling it and I would happily change this if anyone has a better suggestion.

Reply:
I'm curious, would there be value in not rounding the other metrics too? Or to rounding them all to a set number of decimal places (say 2 or 3)?

Reply:
Good question! I think most (all?) of the other metrics are timestamps in nanoseconds? If so it probably wouldn't hurt to remove rounding or set to 2/3 decimal places as you suggest. I wonder why the rounding was added in here in the first place?

Reply:
If we need to do integers, it'd be more effective to multiply the CLS value by 100 and round